### ISSUE TITLE
error at login with patch 10.0

### ISSUE THREAD

**Issue by Daeveren** [Kanegasi/ExaltedPlus#1]
27x Interface/AddOns/ExaltedPlus/epcore.lua:11: attempt to index field 'paragonFramesPool' (a nil value)
[string "@Interface/AddOns/ExaltedPlus/epcore.lua"]:11: in function `update'
[string "@Interface/AddOns/ExaltedPlus/epcore.lua"]:41: in function <Interface/AddOns/ExaltedPlus/epcore.lua:41>
[string "=[C]"]: in function `UpdateBarsShown'
[string "@Interface/FrameXML/StatusTrackingManager.lua"]:143: in function <Interface/FrameXML/StatusTrackingManager.lua:129

**Comment by arcadiatostark** [comment]
```
127x Interface/AddOns/ExaltedPlus/epcore.lua:11: attempt to index field 'paragonFramesPool' (a nil value)
[string "@Interface/AddOns/ExaltedPlus/epcore.lua"]:11: in function `update'
[string "@Interface/AddOns/ExaltedPlus/epcore.lua"]:41: in function <Interface/AddOns/ExaltedPlus/epcore.lua:41>
[string "=[C]"]: in function `UpdateBarsShown'
[string "@Interface/FrameXML/StatusTrackingManager.lua"]:143: in function <Interface/FrameXML/StatusTrackingManager.lua:129>
[string "=[C]"]: ?
[string "=[C]"]: in function `RegisterCVar'
[string "@Interface/SharedXML/CvarUtil.lua"]:2: in function `RegisterCVar'
[string "@Interface/AddOns/InstancePortals/IPCore.lua"]:23: in function `InstancePortalUI_OnEvent'
[string "*InstancePortals.xml:13_OnEvent"]:1: in function <[string "*InstancePortals.xml:13_OnEvent"]:1>

Locals:
_ = nil
id = nil
v = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = "attempt to index field 'paragonFramesPool' (a nil value)"
faction = <table> {
}
f = Frame {
 0 = <userdata>
 update = <function> defined @Interface/AddOns/ExaltedPlus/epcore.lua:10
 c = <table> {
 }
 enumfactions = <function> defined @Interface/AddOns/ExaltedPlus/epcore.lua:2
}

```

**Comment by kanegasi** [comment]
fixed in version 10.0.0-1
